book/src/bridge/extern_rustqt.md

```diff
@@ -30,6 +30,8 @@ A bridge module may contain zero or more `extern "RustQt"` blocks.
 This complements the [`extern "Rust"` CXX section](https://cxx.rs/extern-rust.html)
 but allows for declaring Qt specific features on C++ types.
 
+Automatically converting to camel or snake case can be done through an [attribute](./attributes.md#automatic-case-conversion) for "RustQt" and "C++Qt" blocks.
```

Comment:
I would remove the `for "RustQt" and "C++Qt" blocks` as this page is specifically for `RustQt` blocks anyway and then the attribute page explains which blocks this works for.